components/phpunit_bridge.rst

```diff
@@ -24,6 +24,14 @@ It comes with the following features:
 * Provides a modified version of PHPUnit that does not embed ``symfony/yaml`` nor
   ``prophecy`` to prevent any conflicts with these dependencies.
 
+* Provides polyfills for methods that are not available in older version of
+  PHPUnit
+
+* Provide namespaced class name for older version of PHPUnit
+
+* Makes compatible testSuite with both PHPUnit 8 and previous version by
```

Comment:
```suggestion
* Makes test suite compatible for PHPUnit 8 and the previous version by
```

Reply:
Is able to remove the `void` return type on `setUp`/`tearDown` methods so that PHPUnit 8 can be used with test suites that must run on PHP < 7.1;